app/views/shared/_equipment_landing.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="EquipoGrid">
+  <div class="row">
+    <% @equipments.each do |e| %>
+        <div class="col-4">
+          <div class="EquipoCard" data-id="<%= e.id %>" data-lid="<%= e.lab_space.lab.id %>" data-lsid="<%= e.lab_space.id %>">
+            <div class="icon printer center-icon"></div>
+            <p class="name"><%= link_to e.name, lab_lab_space_equipment_path(e.id, e.lab_space.id, e.lab_space.lab.id) %></p>
```

Comment:
I think `e.lab_space.id` and `e.lab_space.lab.id` are not in the correct order.

Reply:
Nice